internal/kokoro/vet.sh

```diff
@@ -8,24 +8,27 @@ set -x
 
 # Only run the linter on go1.11, since it needs type aliases (and we only care about its output once).
```

Comment:
s/go1.11/go1.12/

in comment

or say "only run the linter on the most recent version"

Reply:
Done